### PR TITLE
Fleshed out math unit tests before adding SIMD implementations, added (c)uint support to Arithmetic

### PR DESCRIPTION
**Pull request by ncorgan**

### CODE REVIEW

File: math/TestConjugate.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        POTHOS_TEST_TRUE(topology.waitInactive(0.01));
+    }
+
+    auto outputs = sink.call<Pothos::BufferChunk>("getBuffer");
+    POTHOS_TEST_EQUAL(expectedOutputs.dtype, outputs.dtype);
+    POTHOS_TEST_EQUAL(expectedOutputs.elements(), outputs.elements());
+    POTHOS_TEST_EQUALA(
+        expectedOutputs.as<const Type*>(),
+        outputs.as<const Type*>(),
+        bufferLen);
```
Comment (ncorgan) [inline math/TestConjugate.cpp]: Test with ComplexType

File: math/TestTrigonometric.cpp [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+#include <Pothos/Framework.hpp>
+#include <Pothos/Testing.hpp>
+
+#include <algorithm>
+#include <cstring>
+#include <iostream>
+#include <vector>
+
+#define _USE_MATH_DEFINES
```
Comment (guruofquality) [inline math/TestTrigonometric.cpp]: Pothos/Config.hpp actually sets this so we dont have to worry about sprinkling _USE_MATH_DEFINES around for windows

File: math/TestConjugate.cpp (hunk 1 above)
Comment (ncorgan) [reply math/TestConjugate.cpp]: Fixed.

File: math/TestTrigonometric.cpp (hunk 2 above)
Comment (ncorgan) [reply math/TestTrigonometric.cpp]: Fixed.

**Pull request merged**